packages/typo3-docs-theme/assets/sass/_highlightjs_github.scss

```diff
@@ -11,7 +11,7 @@
 
 .hljs {
   color: #24292e;
-  background: #ffffff;
+  //background: #ffffff; // The code-block has already a gray background set!
```

Comment:
Many style checkers for scss dissallow comented out code..

Reply:
Ours not. I want to keep it in case we update that file (it is from highlightjs) and have a comment about it. That way it is not added again then.